src/App.js

```diff
@@ -38,35 +38,49 @@ const UserIsAuthenticated = UserAuthWrapper({
  * Expose App base component. Handle all routes
  */
 
+const buildRoutes = (store, onLogPageView, defaultRoute, features, userManager) => {
+
+  const getComponent = component => features.authEnabled ? UserIsAuthenticated(component) : component;
+
+  const router = (
+    <Router history={browserHistory} onUpdate={onLogPageView}>
+      <Redirect from="/" to={defaultRoute} />
+      <Route path="login" component={Login} />
+      <Route path="about" component={About} />
+      <Route path="/callback" component={CallbackPage} />
+      {features.trust !== false ?
+        <div>
+          <Route path="search-creator" component={SearchCreator} />
+          <Route path="saved-searches" component={SeeAllSearches}/>
+          <Route path="saved-search/:id" component={SearchDetail} />
+          <Route path="edit-search/:id" component={SearchEditor} />
+        </div>
+      : null}
+      {features.ask ?
+        <div>
+          <Route path="forms" component={getComponent(FormList)} />
```

Comment:
I like this idea, but we should punt on it for now in the interest of getting this out the door. I'd definitely like to sit with you @okbel and figure out a cleaner way to do this. Does everything else work for you otherwise?